src/components/header/Header.tsx

```diff
@@ -9,11 +9,13 @@ import useMediaQuery from '../../data/hooks/UseMediaQuery';
 import { Text } from '../common/Typography';
 import ConnectWalletButton from './ConnectWalletButton';
 import { IS_DEV } from '../../util/Env';
+import { useAccount } from 'wagmi';
```

Comment:
I don't love the idea of having this hook in the Header component, but I guess it makes sense if we want to hide the Portfolio tab unless a wallet is connected